Rakefile

```diff
@@ -67,7 +67,7 @@ task :cross_compile => :release_prep do
   puts "Cross compiling packages."
   puts "#################################\n\n\n"
 
-  @ldflags = %{-X main.CanaryVersion '#{@release_version}'}
+  @ldflags = %{-X main.CanaryVersion='#{@release_version}'}
```

Comment:
I think it broke after I upgraded to go1.8, which perhaps is more aggressive. But I'm not sure.

I think I also have a PR which switches to using `go` itself to cross-compile (appcanary/agent#152)